stdlib/public/core/StringCharacterView.swift

```diff
@@ -344,6 +341,14 @@ extension String.CharacterView : BidirectionalCollection {
       //   0xAC00–0xD7AF
       case 0xac00...0xd7af: return true
 
+      // Common general use punctuation, excluding extenders:
+      //   0x2010-0x2029
+      case 0x2010...0x2029: return true
+
+      // CJK punctuation characters, excluding extenders:
```

Comment:
The characters (i.e. scalars) that have the "extend" property for the purposes of grapheme breaking. Such characters usually* don't have a grapheme break before them. E.g. 0x302A.

See:
http://unicode.org/reports/tr29/#GB9
http://www.unicode.org/Public/UCD/latest/ucd/auxiliary/GraphemeBreakProperty.txt